### PR TITLE
chore: readme when backup files

### PR DESCRIPTION
**Pull request by yuqizhou77**
- Generate readme when backing up files.

### CODE REVIEW

File: packages/fx-core/resource/backup-config-and-state-change-logs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Congratulations! You are now switching to another tenant or subscription. Configuration files for last provision/local debug are saved successfully.
+
+You are now using a different Microsoft 365 account(M365) or Azure subscription for the provision or local debug from what you previously used. We have backed up the configuration files which are used for or generated from previous provision/local debug. For more information, you can check this [doc](https://aka.ms).
+
```
Comment (yuqizhou77) [inline packages/fx-core/resource/backup-config-and-state-change-logs.md:4]: TODO: will replace the link of the correct doc later.

File: docs/fx-core/switch-account-or-subscription-help.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Note: if you want to recover for a remote environment and you have added new features, please edit the value of "provisionSucceeded" to "false" to provision resources required for the newly added features.
+* If `{env}.{time}.userdata` exists in the backup folder, replace the content of `{env}.userdata` with the content of `{env}.{time}.userdata`. 
+* If you want to recover for a remote environment and your project previously contains Azure sources, update the value of "resourceBaseName" and "botServiceName"(delete this key if not exists) to the value defined in `azure.parameters.{env}.{time}.json`.
+* Run provision and deploy again.    
+* Delete the backups when you think there is no need to keep them.
+
+## Error
+### Could not be redirected to the teams web page correctly
+If you have previewed (local or remote) your Teams app in one M365 tenant and then switch to another M365 account, you may encounter errors once the browser is launched when previewing in the new M365 tenant. This happens due to the account info saved in the browser storage.
+
```
Comment (MuyangAmigo) [inline docs/fx-core/switch-account-or-subscription-help.md:65]: Probably add a screenshot here? Seems hard to understand this error just based on description.

File: docs/fx-core/switch-account-or-subscription-help.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+## Switch Microsoft 365 account
+### Local debug
+You could run local debugging for a Teams project with one M365 account and then easily switch to another account for further local debugging. To do this, you only need to:
+1. Sign out of the current M365 account.
+2. Sign in to the new account.
+3. Start local debugging.
+
+After that, we will 
+1. Back up configuration files for local environment. [Learn more about backup & recover](#backup--recover).
```
Comment (MuyangAmigo) [inline docs/fx-core/switch-account-or-subscription-help.md:13]: Can we be more explicit on what exactly are the files that will be backed up? People have different understanding on `configuration files`.

File: docs/fx-core/switch-account-or-subscription-help.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* Run provision and deploy again.    
+* Delete the backups when you think there is no need to keep them.
+
+## Error
+### Could not be redirected to the teams web page correctly
+If you have previewed (local or remote) your Teams app in one M365 tenant and then switch to another M365 account, you may encounter errors once the browser is launched when previewing in the new M365 tenant. This happens due to the account info saved in the browser storage.
+
+#### Mitigation
+* Launch browser in incognito mode    
+runtimeArgs are the arguments passed to the runtime executable. It is possible to start the browser in private mode. You could edit the launch configuration by adding `"runtimeArgs": ["--inprivate"]` (for Edge) or `"runtimeArgs": ["--incognito"]` (for Chrome).
```
Comment (MuyangAmigo) [inline docs/fx-core/switch-account-or-subscription-help.md:68]: Can we give a more complete example of `launch.json`, developers who's not that familiar with VSC probably won't just get it where to add these two arguments.

File: packages/fx-core/resource/backup-config-change-logs.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Congratulations! You are now switching to another tenant or subscription. Configuration files for last provision/local debug are saved successfully.
+
+You are now using a Microsoft 365 account or an Azure subscription (applies only when provisioning in a remote environment of a project requiring cloud resources) that is different from what you used previously in the selected environment for provision or local debugging. We have backed up the configuration files which are used for or generated from previous provision or local debug. For more information, you can check this [doc](https://aka.ms).
+
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/backup-config-change-logs.md:4]: The aka link isn't created?

File: packages/fx-core/resource/backup-config-change-logs.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+## Why backup
+Configuration files will be overwritten by Teams Toolkit when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365 account. We will back up those files so that you could continue using the resources created before easily when you decide to switch back to the accounts or the subscription that you selected before. Otherwise, new resources will be created, and you have to delete the old resources manully to avoid costs.
+
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/backup-config-change-logs.md:7]: Another reason we do backup besides "recover" is to help developers locate the resources we created e.g. the Teams app id so they could delete them manually, Probably we can mention that as well.

File: packages/fx-core/resource/backup-config-change-logs.md (hunk 5 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/backup-config-change-logs.md:4]: Yes, will be created after this PR is completed, and then update the link

File: docs/fx-core/switch-account-or-subscription-help.md (hunk 3 above)
Comment (yuqizhou77) [reply docs/fx-core/switch-account-or-subscription-help.md:13]: the detailed files are described in the link. [Learn more about backup & recover](#backup--recover).

File: packages/fx-core/resource/backup-config-change-logs.md (hunk 5 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/backup-config-change-logs.md:4]: Created link. but the link will only work after this PR is checked in.

**Review by MuyangAmigo — APPROVED** [review]
Most of my comments are addressed, so approve for docs part

**Review by jayzhang — APPROVED** [review]

**Pull request merged**